docker/docs/Dockerfile

```diff
@@ -9,21 +9,31 @@ FROM ${PARENT_IMAGE}:${IMAGE_VERSION} AS pecandocs
 
 RUN apt-get update \
     && apt-get install -y --no-install-recommends pandoc \
-    && Rscript \
-        -e 'repos <- c(getOption("repos"),' \
-        -e '           sub(r"(\d{4}-\d{2}-\d{2})", "latest", getOption("repos")))' \
-        -e 'remotes::install_version("rmarkdown", ">= 2.19", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
-        -e 'remotes::install_version("knitr", ">= 1.42", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
-        -e 'remotes::install_version("bookdown", ">= 0.31", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
+    && Rscript -e 'repos <- c(getOption("repos"), sub(r"(\d{4}-\d{2}-\d{2})", "latest", getOption("repos"))); \
+                   remotes::install_version("rmarkdown", ">= 2.19", dependencies = TRUE, upgrade = FALSE, repos = repos); \
+                   remotes::install_version("knitr", ">= 1.42", dependencies = TRUE, upgrade = FALSE, repos = repos); \
+                   remotes::install_version("bookdown", ">= 0.31", dependencies = TRUE, upgrade = FALSE, repos = repos); \
+                   install.packages("pkgdown")' \
     && rm -rf /var/lib/apt/lists/*
 
+# ----------------------------------------------------------------------
+# Compile bookdown documentation
+# ----------------------------------------------------------------------    
 WORKDIR /src/book_source/
 COPY book_source /src/book_source/
 COPY documentation /src/documentation/
 COPY docker-compose.yml /src/
 COPY docker/env.example /src/docker/
 RUN make build
 
+# ----------------------------------------------------------------------
+# Build pkgdown documentation
+# ----------------------------------------------------------------------
+WORKDIR /src/
+COPY package_documentation /src/package_documentation
+COPY Makefile /src/
+RUN make pkgdocs
+
```

Comment:
Paging @robkooper for a best practice question: Better to build both the book and the pkgdown pages inside the same temporary container (as currently implemented), or to put the pkgdown build into a separate `FROM ... AS pecanpkgdocs`?